test_notebooks/trajectory_visualization.py

```diff
@@ -39,44 +39,25 @@ def download_trajectory_data(file):
         return (file_csv, file_exists)
 
 
-def sample_data():
+def csv_writer():
     """
-    Generates sample .csv file for unit tests.
+    Creates a sample dataset for unit tests called output.csv
```

Comment:
Does this unit test variable have to be defined in this file, or would it still work if defined in the unit test file?